src/CalculatorUITests/UnitConverterFunctionalTests.cs

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using CalculatorUITestFramework;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OpenQA.Selenium;
+using System;
+
+namespace CalculatorUITests
+{
+    [TestClass]
+    public class UnitConverterFunctionalTests
```

Comment:
Nit: For organization purposes, can we call these CurrencyConverterFunctionalTests?  If we end up adding tests for each mode, it will help keep things organized.

Reply:
This is done.